app/helpers/fuzzy_time.rb

```diff
@@ -22,9 +23,9 @@ def ago(timestamp)
                      when less_than(23.5 * hours)
                        "about #{(diff / (1 * hours)).round} hours ago"
                      end
-          "<span data-toggle='tooltip' title='#{timestamp.strftime('%e %B %Y at %H:%M %Z')}'>#{response}</span>"
+          "<span data-toggle='tooltip' data-title='#{formatted_date}'>#{response}</span>"
         else
-          timestamp.strftime("%e %B %Y at %H:%M %Z")
+          "<span class='localize-time'>#{formatted_date}</span>"
         end
```

Comment:
The problem was that I was trying to save a step... I made one check for the timezone abbreviation and then stored and used it for all the dates. Obviously this wasn't a good idea. :| 

I'm pushing a new version now.